glib-sys/src/lib.rs

```diff
@@ -645,30 +648,7 @@ pub const G_DATALIST_FLAGS_MASK: c_int = 3;
 pub const G_DATE_BAD_DAY: c_int = 0;
 pub const G_DATE_BAD_JULIAN: c_int = 0;
 pub const G_DATE_BAD_YEAR: c_int = 0;
-pub const G_DIR_SEPARATOR: c_int = 92;
-pub const G_DIR_SEPARATOR_S: *const c_char = b"\\\0" as *const u8 as *const c_char;
 pub const G_E: c_double = 2.718282;
-pub const G_GINT16_FORMAT: *const c_char = b"hi\0" as *const u8 as *const c_char;
-pub const G_GINT16_MODIFIER: *const c_char = b"h\0" as *const u8 as *const c_char;
-pub const G_GINT32_FORMAT: *const c_char = b"i\0" as *const u8 as *const c_char;
-pub const G_GINT32_MODIFIER: *const c_char = b"\0" as *const u8 as *const c_char;
-//pub const G_GINT64_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINT64_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-pub const G_GNUC_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
-pub const G_GNUC_PRETTY_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
-//pub const G_GSIZE_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
-//pub const G_GSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-//pub const G_GSSIZE_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GSSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-pub const G_GUINT16_FORMAT: *const c_char = b"hu\0" as *const u8 as *const c_char;
-pub const G_GUINT32_FORMAT: *const c_char = b"u\0" as *const u8 as *const c_char;
-//pub const G_GUINT64_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
-//pub const G_GUINTPTR_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
-pub const G_HAVE_GINT64: c_int = 1;
-pub const G_HAVE_GNUC_VARARGS: c_int = 1;
-pub const G_HAVE_GNUC_VISIBILITY: c_int = 1;
 pub const G_HAVE_GROWING_STACK: c_int = 0;
 pub const G_HAVE_ISO_VARARGS: c_int = 1;
```

Comment:
Should go away, these two. basically all G_HAVE_.* and G_.*FORMAT and G_.*MODIFIER